jest.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
   // clearMocks: false,
 
   // Indicates whether the coverage information should be collected while executing the test
-  collectCoverage: true,
+  collectCoverage: false,
 
```

Comment:
I stopped displaying the coverage reports when running the tests; they were getting cumbersome and that's a low priority at this point